docs/389ds/FAQ/bdb-deprecation.md

```diff
@@ -0,0 +1,277 @@
+| mdb                       | dbiEntries-%d                 |
+| mdb                       | dbiFlags-%d                   |
+| mdb                       | dbiLeafPages                  |
+| mdb                       | dbiLeafPages-%d               |
+| mdb                       | dbiName-%d                    |
+| mdb                       | dbiOverflowPages              |
+| mdb                       | dbiOverflowPages-%d           |
+| mdb                       | dbiPageSize                   |
+| mdb                       | dbiPageSize-%d                |
+| mdb                       | dbiTreeDepth                  |
+| mdb                       | dbiTreeDepth-%d               |
+| bdb or mdb                | dnCacheHitRatio               |
+| bdb or mdb                | dnCacheHits                   |
+| bdb or mdb                | dnCacheTries                  |
+| bdb or mdb                | entryCacheHitRatio            |
+| bdb or mdb                | entryCacheHits                |
+| bdb or mdb                | entryCacheTries               |
+| mdb                       | grantTimeROtxn                |
+| mdb                       | grantTimeRWtxn                |
+| mdb                       | lifeTimeROtxn                 |
+| mdb                       | lifeTimeRWtxn                 |
+| bdb or mdb                | maxDnCacheCount               |
+| bdb or mdb                | maxDnCacheSize                |
+| bdb or mdb                | maxEntryCacheCount            |
+| bdb or mdb                | maxEntryCacheSize             |
+| bdb or mdb                | maxNormalizedDnCacheSize      |
+| bdb or mdb                | NormalizedDnCacheEvictions    |
+| bdb or mdb                | normalizedDnCacheHitRatio     |
+| bdb or mdb                | normalizedDnCacheHits         |
+| bdb or mdb                | normalizedDnCacheMisses       |
+| bdb or mdb                | NormalizedDnCacheThreadSize   |
+| bdb or mdb                | NormalizedDnCacheThreadSlots  |
+| bdb or mdb                | normalizedDnCacheTries        |
+| bdb or mdb                | readOnly                      |
+| mdb                       | waitingROtxn                  |
+| mdb                       | waitingRWtxn                  |
+
+### Performances impact
+
+Several point related to the implementations differences between lmdb and bdb
+
+impacts the performances
+
+[A] lmdb  does not support DB_RECNUM database.
+This impacts the VLV searches (there is no fast way to get a record by using its position,
+  we have to traverse all the records and count them so even if VLV searches are still working with lmdb, they are very slow.
+
+[B] The locking mechanism is very different. There can only be a single write transaction at a given time.
+
+The fact that the writes needs to be serialized  impacts:
+
+- The import performances because the database write needs to be serialized.
+
+- The multiple backends write performances
```

Comment:
Just out of curiosity, does that mean we will have frequent deadlock with parallel updates  (trimming retroCL and direct update) ?

Reply:
No deadlocks but lock contention. ( thread will wait to be able to start the txn )
For the cl trimming that means that the number of csn that are looked/up or removed in a single txn is limited.
 (So that direct updates may have a chance to get the txn lock)